content/components/banner.mdx

```diff
@@ -8,15 +8,254 @@ tags:
+  </Box>
+</Box>
+
+### Dismissable
+
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 4}}
+>
+  <img
+    width="456"
+    alt="banners with a dismiss button"
+    src="https://github.com/primer/design/assets/813754/e9863c48-cf3a-472b-a066-0ba139a633ca"
+  />
+  <Box as="p" mt="0">
+    <h4>Dismiss button</h4>
+    Banners with an <i>onDismiss</i> property automatically show a dismiss button.
+  </Box>
+</Box>
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 4}}
+>
+  <img
+    width="456"
+    alt="a critical banner with no actions and one with a single action"
+    src="https://github.com/primer/design/assets/813754/2463e14c-5cb9-4bb4-9e3a-474c60c32da4"
+  />
+  <Box as="p" mt="0">
+    <h4>Critical banner</h4>
+    <p>
+    Critical banner can't be made dismissable. Since they should only be used for errors that need to be resolved, the only way to get rid of a critical banner is to resolve the error. If they are used to warn the user about destructive actions the banner should also not be dismissable.</p>
+    <p>
+    If the error must be resolved on a different page, it is recommended to add an action that directs the user to the page where the error can be resolved.
+    </p>
+  </Box>
+</Box>
+
+### Actions
+
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 4}}
+>
+  <img
+    width="456"
+    alt="trailing actions on a banner"
+    src="https://github.com/primer/design/assets/813754/b4d2e88e-2f0e-4b29-8f21-13ac8399c943"
+  />
+  <Box as="p" mt="0">
+    <h4>Actions</h4>
+    <p>A maximum of 2 actions can be added to a banner. Whenever possible it is recommended to only add a single action to better direct the users attention.</p>
+    <p>You can choose between adding a primary or secondary action, depending on how much attention you want to draw to the action.</p>
+    <p>If you do need two actions in your banner, one of them must be primary and the other must be secondary. In this case, use the primary action for the recommended action for example, to enable a feature and the secondary one to get more information.</p>
+  </Box>
+</Box>
+
+### Icon
+
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 4}}
+>
+  <img
+    width="456"
+    alt="banners with icons, some of which are customized"
+    src="https://github.com/primer/design/assets/813754/dd6d9159-8de7-4f71-b69b-ede15284aeb4"
+  />
+  <Box as="p" mt="0">
+    <h4>Icon</h4>
+    The icon relates to the selected <a href="#variant">variant</a> and can't be disabled. It can only be changed for banners of variant <i>info</i> or <i>upsell</i>.
```

Comment:
```suggestion
    The icon relates to the selected <a href="#variant">variant</a> and can't be disabled. It can only be changed for banners of variant `info` or `upsell`.
```

Reply:
Same as above, how do I get this to work inside of box?

Reply:
Aw, I thought it would work in `Box`. I guess italics is fine 🤷

Reply:
@mperrotti  helped me. Apparently you need blank lines inside the box and the content can't be indented to be recognized as markdown.